src/v2/guide/migration.md

```diff
@@ -726,38 +726,40 @@ to:
 </div>
 {% endraw %}
 
-## Transitions
+## Transiciones
 
-### `transition` Attribute <sup>replaced</sup>
+### Atributo `transition` <sup>reemplazado</sup>
 
-Vue's transition system has changed quite drastically and now uses `<transition>` and `<transition-group>` wrapper elements, rather than the `transition` attribute. It's recommended to read the new [Transitions guide](transitions.html) to learn more.
+El sistema de transición de Vue ha cambiado drásticamente y ahora usa elementos de envoltura `<transition>` y `<transition-group>`, en lugar del atributo `transition`. Se recomienda leer la nueva [Guía de transiciones](transitions.html) para obtener más información.
 
 {% raw %}
 <div class="upgrade-path">
-  <h4>Upgrade Path</h4>
-  <p>Run the <a href="https://github.com/vuejs/vue-migration-helper">migration helper</a> on your codebase to find examples of the <code>transition</code> attribute.</p>
+  <h4>Ruta de actualización</h4>
+  <p>Ejecute el <a href="https://github.com/vuejs/vue-migration-helper">asistente de migración</a> en su código para encontrar ejemplos del atributo <code>transition</code>.</p>
 </div>
 {% endraw %}
 
-### `Vue.transition` for Reusable Transitions <sup>replaced</sup>
+### `Vue.transition` para transiciones reutilizables <sup>reemplazado</sup>
 
-With the new transition system, you can now [use components for reusable transitions](transitions.html#Reusable-Transitions).
+Con el nuevo sistema de transición, ahora puede [usar componentes para transiciones reutilizables](transitions.html#Reusable-Transitions).
 
 {% raw %}
 <div class="upgrade-path">
-  <h4>Upgrade Path</h4>
-  <p>Run the <a href="https://github.com/vuejs/vue-migration-helper">migration helper</a> on your codebase to find examples of <code>Vue.transition</code>.</p>
+  <h4>Ruta de actualización</h4>
+  <p>Ejecute el <a href="https://github.com/vuejs/vue-migration-helper">asistente de migración</a> en su código para encontrar ejemplose de <code>Vue.transition</code>.</p>
 </div>
 {% endraw %}
 
-### Transition `stagger` Attribute <sup>removed</sup>
+### Transición atributo `stagger` <sup>removido</sup>
```

Comment:
This looks a bit odd to me 😞, but I can't think of a better way to translate it 😢

Reply:
Yes, it's a hard one. Open for discussion I guess...

Reply:
@Tintef @FLarra  Checking [the portuguese translation](https://br.vuejs.org/v2/guide/migration.html#Atributo-de-Transicao-stagger-removido), they are using `Atributo de Transición`. What do you think?

Reply:
I like it. Let me change it. Thank you!